libs/api/domains/occupational-licenses/src/lib/models/occupationalLicense.model.ts

```diff
@@ -32,18 +40,18 @@ export abstract class OccupationalLicense {
   institution!: OccupationalLicenseType
 
   @Field(() => ID)
-  id!: string
+  id!: string | number
   @Field(() => String)
   type!: string
 
   @Field(() => String)
   profession!: string
 
-  @Field(() => Boolean)
-  isValid!: boolean
-
   @Field(() => String)
   validFrom!: string
+
+  @Field(() => String)
+  isValid!: Validity
```

Comment:
```suggestion

  @Field(() => Validty)
  isValid!: Validity
```

Why not make a graphql enum `Validity` (with registerEnumType) and typecheck it here so you don't need to do the `license.isValid as Validity`